apps/ubuntu_bootstrap/test/storage/recovery_key/test_recovery_key.dart

```diff
@@ -10,5 +10,6 @@ export 'test_recovery_key.mocks.dart';
 RecoveryKeyModel buildRecoveryKeyModel({bool? init}) {
   final model = MockRecoveryKeyModel();
   when(model.init()).thenAnswer((_) async => init ?? false);
+  when(model.confirmed).thenReturn(true);
```

Comment:
Not sure if it's worth having a minimal test that checks whether or not the 'next' button is enabled depending on the checkobox state.

Reply:
Maybe, but the logic is literally just a direct check of the boolean value so all it would be testing is that the checkbox's value is `model.confirmed` :shrug: 
```dart
          NextWizardButton(
            enabled: ref.watch(
              recoveryKeyModelProvider.select((model) => model.confirmed),
            ),
          ),
```